xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -612,6 +640,12 @@ bool CPVRTimerInfoTag::UpdateEntry(const CPVRTimerInfoTagPtr &tag)
   if (m_strSummary.empty())
     UpdateSummary();
 
+  m_iActiveChildTimers = 0;
+  m_bHasChildConflictNOK = false;
+  m_bHasChildRecording = false;
+  m_bHasChildErrors = false;
+  // Updated in CPVRTimers::UpdateEntries which should run after this...
```

Comment:
I agree.
I've kept a comment referring to UpdateChildState as well as adding a description for both fucntions and placing them next to eachother in PVRTimerInfoTag.h.
If you have any better ideas, I'm sure you will let me know.